tests/test_utils.py

```diff
@@ -184,7 +183,7 @@ def test_get_thresholds_value_error(image_random: np.ndarray) -> None:
 
 def test_create_empty_dataframe() -> None:
```

Comment:
This only tests the `grainstats_df` dataframe is correctly created. I will parametrise the test to check all three possible options (also makes it easier to extend if/when newer empty dataframes are required).